examples/visibility_ingest_cookbook/config_winchefclient/LICENSE

```diff
@@ -0,0 +1,3 @@
+Copyright 2016, All Rights Reserved.
```

Comment:
Our cookbook has Apache 2 only, lets remove that here, since we have a LICENSE in our project root